layouts/partials/author-date.html

```diff
@@ -0,0 +1,3 @@
+{{ with .Param "author" }}<span class="author" itemprop="author">{{ . }}</span>{{ end -}}
+{{ if and .Date (.Param "author") }} - {{ end -}}
+{{ with .Date }}<time class="created-date" datetime="{{ .Format "2006-01-02T15:04:05Z07:00" }}">{{ .Format ($.Param "dateformat" | default "2 January, 2006") }}</time>{{ end }}
```

Comment:
Can we change it back to "$.Param", see https://gohugo.io/functions/param/#prose

Otherwise nice improvement!

Reply:
I didn't mean to change this. Thanks for spotting that! I pushed a fixup commit.